crazyflie/launch/launch.py

```diff
@@ -66,15 +61,15 @@ def generate_launch_description():
 
     return LaunchDescription([
         DeclareLaunchArgument('backend', default_value='cpp'),
-        DeclareLaunchArgument('rviz', default_value='show'),
```

Comment:
The current script adds rviz as an option so as to launch it seperately, crazyswarm2 do not have a proper map visualization module